src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+- **Application Users**: these people interact with applications on the Ethereum blockchain;
+- **Ether holders**: these people own the ETH token.
+
+_Note: any individual can be part of multiple of these groups (e.g. a protocol developer could champion an EIP, and run a beacon chain validator, and use DeFi applications). For conceptual clarity, it is easiest to distinguish between them, though._
+
+<Divider />
+
+## What is an EIP? {#what-is-an-eip}
+
+One important process used in Ethereum Governance is the proposal of **Ethereum Improvement Proposals (EIPs)**. EIPs are standards specifying potential new features or processes for Ethereum. Anyone within the Ethereum community can create an EIP. For example, none of the authors of EIP-721, the EIP that standardized NFTs, have worked directly on Ethereum's protocol development.
+
+<ButtonLink to="/eips/">More on EIPs</ButtonLink>
+
+<Divider />
+
+## The Formal Process {#formal-process}
+
+The formal process for introducing changes to the Ethereum protocol is as follows:
+
+1. **Propose a Core EIP**: as described in [EIP-1](https://eips.ethereum.org/EIPS/eip-1#core-eips), the first step to formally proposing a change to Ethereum is to detail it in a Core EIP. This will act as the official specification for an EIP that Protocol Developers will implement if accepted.
+
+2. **Present your EIP to Protocol Developers**: once you have a Core EIP, you should get feedback on it from Protocol Developers. You can get feedback can by proposing it for discussion on an [AllCoreDevs call](https://github.com/ethereum/execution-specs/tree/master/network-upgrades#getting-the-considered-for-inclusion-cfi-status). Alternatively, discussion can happen asynchronously on the [Ethereum Magician's forum](https://ethereum-magicians.org/) or in the [Ethereum R&D Discord](https://discord.gg/mncqtgVSVw).
+
+> Potential outcomes of this stage are:
+
+> - The EIP will be considered for a future network upgrade
+> - Technical changes will be requested
+> - It may be rejected if it is not a priority or the improvement is not large enough relative to the development effort.
+
+3. **[Optional] Gather Community Consensus for your EIP**: some EIPs are purely technical changes with little room for debate, but others can be more contentious. If the EIP is in the latter category, Protocol Developers will likely want to see strong evidence of overwhelming community support before proceeding with it. There is no "one size fits all" approach here, as each EIP is unique.
+
+4. **EIP Included in Network Upgrade**: assuming the EIP is approved, tested and implemented, it gets scheduled as part of a network upgrade. Given the high coordination costs of network upgrades (everyone needs to upgrade simultaneously), EIPs are generally bundled together in upgrades.
```

Comment:
I wonder to what degree we should explain how updates occur through forks & what a soft vs. hard fork is 🤔 

At the very least we should probably link to https://ethereum.org/en/history/ which explains forks a bit.

Reply:
Hmm yeah, it feels like this is a tangent but an important one. I think it may be worth adding a paragraph to the "What are forks?" sections of /history which explains Ethereum's preference (and need!) for hard vs. soft forks. 

Vitalik has a good article on this: https://vitalik.ca/general/2017/03/14/forks_and_markets.html 

The most important bits IMO:
* Hard forks are needed to add anything which _expands_  or _changes_ the consensus rules, which is the default case for most EIPs (e.g. add a feature, raise a gas cost);
* Soft forks are only useful if you are strictly restricting the rules of the network (e.g. cap the block gas limit in protocol).